packages/web3/package.json

```diff
@@ -44,7 +44,8 @@
 		"jest-extended": "^1.1.0",
 		"prettier": "^2.4.1",
 		"ts-jest": "^27.0.7",
-		"typescript": "^4.5.2"
+		"typescript": "^4.5.2",
+		"web3-eth-personal": "4.0.0-alpha.0"
```

Comment:
Why web3-eth-personal is added in `devDependencies` in this PR? instead of `dependencies`, and are you using it in main web3 package?